<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaDetailsView</class>
 <widget class="QWidget" name="MediaDetailsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="summaryButton">
     <property name="text">
      <string>summary</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="videoLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>video</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="codecsLabel">
     <property name="text">
      <string>codecs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
